cognite/client/_api/functions.py

```diff
@@ -476,8 +475,8 @@ def _assert_exactly_one_of_folder_or_file_id_or_function_handle(
         folder: str | None, file_id: int | None, function_handle: Callable[..., Any] | None
     ) -> None:
         source_code_options = {"folder": folder, "file_id": file_id, "function_handle": function_handle}
-        given_source_code_options = [key for key in source_code_options.keys() if source_code_options[key]]
-        if len(given_source_code_options) < 1:
+        given_source_code_options = [key for key in source_code_options if source_code_options[key]]
+        if not given_source_code_options:
```

Comment:
So if I make use of `exactly_one_is_not_none`, then I would have to combine these messages since they are currently splitting the space up into three options (not saying this is a bad idea). Maybe saying something like `"Exactly one of the arguments folder, file_id and handle is required, but {given_source_code_options} was provided"`. Does that sound reasonable to you?

Reply:
Btw, this sounds like a separate PR, let's move the discussion there if you chose to continue with this 👍

Reply:
No worries, I will put it on the board.